File: Java/org/mayope/api/response/ListTreesResponse.java
package org.mayope.api.response;

import java.util;

import org.mayope.api.types;

/**
 * A response containing a list of trees.
 */
public class ListTreesResponse {
    private List<Tree> trees;

    public List<Tree> getTrees() {
        return trees;
    }

    public void setTrees(List<Tree> trees) {
        this.trees = trees;
    }
}

File: CSharp/Mayope.Api/Responses/ListTreesResponse.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;

using Mayope.Api.Types;

using Newtonsoft.Json;

namespace Mayope.Api.Responses;
{
    /// <summary>
    /// A response containing a list of trees.
    /// </summary>
    public class ListTreesResponse
    {
        /// <summary>
        /// The UUID of the forest.
        /// </summary>
        [Required]
        [JsonRequired]
        public Guid Forest { get; set; }

        /// <summary>
        /// The trees in the forest, empty if none.
        /// </summary>
        [Required]
        [JsonRequired]
        public IList<Tree> Trees { get; set; }
    }
}
